modular_skyrat/modules/loadouts/loadout_ui/loadout_manager.dm

```diff
@@ -208,10 +208,14 @@
 /// Set [item]'s name to input provided.
 /datum/loadout_manager/proc/set_item_name(datum/loadout_item/item)
 	var/current_name = ""
+	var/current_desc = ""
 	if(INFO_NAMED in owner.prefs.loadout_list[item.item_path])
 		current_name = owner.prefs.loadout_list[item.item_path][INFO_NAMED]
+	if(INFO_DESCRIBED in owner.prefs.loadout_list[item.item_path])
+		current_desc = owner.prefs.loadout_list[item.item_path][INFO_DESCRIBED]
 
 	var/input_name = stripped_input(owner, "What name do you want to give [item.name]? Leave blank to clear.", "[item.name] name", current_name, MAX_NAME_LEN)
+	var/input_desc = stripped_input(owner, "What description do you want to give [item.name]? [MAX_MESSAGE_LEN] character max, leave blank to clear.", "[item.name] description", current_desc, MAX_MESSAGE_LEN)
```

Comment:
TGUI stuff please.

Also, max input for the description of items should be 256, we don't need whole flavor texts in there.

Reply:
256 is a bit low but u da boss

Reply:
512 maybe?